### PR TITLE
[TASK] Add unit test for PartyService

### PR DESCRIPTION
**Pull request by hlubek**
Releases: master, 3.0

### CODE REVIEW

File: Tests/Unit/Domain/Service/PartyServiceTest.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<?php
+namespace TYPO3\Party\Tests\Unit\Domain\Service;
+
+/*                                                                        *
+ * This script belongs to the TYPO3 Flow framework.                       *
```
Comment (aertmann) [inline Tests/Unit/Domain/Service/PartyServiceTest.php]: This script belongs to the TYPO3 Flow package "Party".

**Comment by aertmann** [comment]
Small nitpick, otherwise it looks good

**Comment by hlubek** [comment]
Fixed that (also in other unit tests)

**Comment by aertmann** [comment]
:+1:

**Pull request merged**